virtualization/windowscontainers/container-networking/network-drivers-topologies.md

```diff
@@ -19,23 +19,24 @@ In addition to leveraging the default 'nat' network created by Docker on Windows
   > Multiple NAT networks are supported if you have the Windows 10 Creators Update installed!
 
 - **transparent** – containers attached to a network created with the 'transparent' driver will be directly connected to the physical network through an *external* Hyper-V switch. IPs from the physical network can be assigned statically (requires user-specified ``--subnet`` option) or dynamically using an external DHCP server. 
+  > Note: Due to the below requirement, connecting your container hosts over a transparent network is not supported on Azure VMs.

```

Comment:
Can you ping Aanand on this? If he doesn't respond, leave it as is. But I think there are some VM instances which can support transparent mode (requires MAC spoofing on Hyper-V host)

Reply:
@aanandr Are there any VM instances on Azure that support transparent mode networking?